solutions/swb-app/src/projectRoutes.ts

```diff
@@ -16,4 +28,50 @@ export function setUpProjectRoutes(router: Router, projectService: ProjectServic
       res.send(projects);
     })
   );
+
+  // add user to the project
+  router.post(
+    '/projects/:projectId/users/:userId',
+    wrapAsync(async (req: Request, res: Response) => {
+      processValidatorResult(validate(req.body, AssignUserToProject));
+
+      const userId = req.params.userId;
+      const projectId = req.params.projectId;
+      const groupId = `${projectId}#${req.body.role}`;
+
+      try {
+        const existingUser = await userService.getUser(userId);
+        const isAdmin = existingUser.roles.some((role) => role === PermissionRoles.Admin);
```

Comment:
what Admin role is this? IT Admin or Project Admin?

Reply:
this is IT admin. We are checking that it admin cannot assign project to another it admin

Reply:
should we be more explicit that it is an IT admin instead of just Admin?